### PR TITLE
feat: support new docker compose

### PR DESCRIPTION
**Pull request by douglaz**

### CODE REVIEW

**Review by justinmoon — DISMISSED** [review]

File: docker/tls-download-mutinynet.sh [hunk 1]
```diff
@@ -6,7 +6,10 @@
 
 DOCKER_COMPOSE_FILE=https://raw.githubusercontent.com/fedimint/fedimint/master/docker/full-tls-mutinynet/docker-compose.yaml
 
-if ! [ -x "$(command -v docker-compose)" ]; then
+DOCKER_COMPOSE=docker-compose
+if docker compose version|grep 'Docker Compose' >& /dev/null; then
```
Comment (EthnTuttle) [inline docker/tls-download-mutinynet.sh:10]: not very DRY. might need to start a `lib.sh` file.

**Review by EthnTuttle — APPROVED** [review]
LGTM

**Comment by EthnTuttle** [comment]
Do we know someone that encountered this recently? I'm unsure how to test this manually without some research first.

**Comment by douglaz** [comment]
> Do we know someone that encountered this recently? I'm unsure how to test this manually without some research first.

I tested locally and it works

**Pull request merged**